bootloaders/riotboot_dfu/Makefile

```diff
@@ -0,0 +1,43 @@
+# Default RIOT bootloader
+APPLICATION = riotboot_dfu
+
+# Default testing board
+BOARD ?= samr21-xpro
+
+# Select the boards with riotboot feature
+FEATURES_REQUIRED += riotboot
```

Comment:
Looks like a 

```
FEATURES_REQUIRED += usbus
```

is needed here

Reply:
I'll add `USEMODULE=riotboot_usb_dfu` by default for this application.

Reply:
Done.